src/Components/Layout/LayoutStyled.js

```diff
@@ -0,0 +1,7 @@
+import styled from 'styled-components'
+import { Container, Divider, Button } from 'semantic-ui-react'
+export const LayoutWrapper = styled(Container)`
+  height: 100vh;
+  display: flex !important;
+  width: 100% !important;
+`
```

Comment:
Generally speaking, I'm not a huge fan of `!important`. Perhaps, instead of overwriting the `.ui.container` styles, we can introduce a wrapper element between the container and the contents that will apply the flex style?
Not a big deal either way.

Reply:
The important is used here to override the defaults of semantic ui.  The scope of !important is only for this component, so there is no global pollution